src/core/array.d

```diff
@@ -0,0 +1,61 @@
+module core.array;
+
+/++
+    Returns a static array constructed from `a`. The type of elements can be
+    specified implicitly (`int[2] a = staticArray(1,2)`) or explicitly
+    (`float[2] a = staticArray!float(1,2)`).
++/
+pragma(inline, true) U[T.length] staticArray(U = CommonType!T, T...)(T a)
+{
+    return [a];
+}
+
+unittest
```

Comment:
Make the unittest public, s.t. it's visible on the docs.

Reply:
Also add `@nogc @safe nothrow and pure`. A static array should be usable in the full set.

Reply:
> Make the unittest public, s.t. it's visible on the docs.
done